modules/ds_bean_fields/ds_bean_fields.module

```diff
@@ -133,6 +139,7 @@ function _ds_bean_fields_render_html($field, $html) {
  * Hyphenate a text string.
  */
 function ds_bean_fields_hyphenate($text, $langcode = NULL) {
+  $patterns = NULL;
```

Comment:
ah, I missed that reading through it. Not sure what that is for either. It has a smell to it. For another day perhaps.